tests/support/loadJsdom.ts

```diff
@@ -31,6 +31,13 @@ global.requestAnimationFrame = (cb: (...args: any[]) => {}) => {
 
 global.cancelAnimationFrame = () => {};
 
+global.IntersectionObserver = function () {
```

Comment:
We should transition to test-extras loadJsdom.  I am not sure why we haven't moved in widget-core.  Why do we have to do this as well?